angular-ngrx-scss/src/app/app.module.ts

```diff
@@ -15,6 +16,7 @@ import { TokenInterceptor } from './auth/services/token.interceptor';
 @NgModule({
   imports: [
     BrowserModule,
+    BrowserAnimationsModule,
```

Comment:
Yes, though it doesn't "technically" have anything to do with the auth flow. lol The nav-bar uses a transition when you toggle the dropdown open and closed. I happened to notice the error for this while testing other things and since it was just this change, went ahead and added it in. :)